app/services/support_interface/send_ucas_match_reminder_emails.rb

```diff
@@ -0,0 +1,40 @@
+module SupportInterface
+  class SendUCASMatchReminderEmails
+    attr_accessor :ucas_match
+
+    def initialize(ucas_match:)
+      @ucas_match = ucas_match
+    end
+
+    class << self
+      def call(**args)
+        new(args).call
+      end
+    end
```

Comment:
Nitpicking a little bit but I think it would be good to keep the ucas email services consistent and not use class methods as they don't add value.